test/postal/test/message.clj

```diff
@@ -312,3 +312,13 @@
             :subject "Test"
             :body "Test!"})]
     (is (.contains m "From: from@bar.dom"))))
+
+(deftest test-make-jmessage
```

Comment:
Great, though looking at it now, I think we can just add `:Content-Type` to `test-extra-headers` and refactor that a bit?

Reply:
Makes sense! I refactored the unit test. And I updated clojure 1.10.2 version too.